src/pages/ResearcherConsole.js

```diff
@@ -364,6 +386,14 @@ class ResearcherConsole extends Component {
           action: { label: "Yes", handler: this.dialogHandlerCancelDAR }
         }, [div({ className: "dialog-description" }, ["Are you sure you want to cancel this Data Access Request?"]),]),
 
+        ConfirmationDialog({
+          title: 'Revise saved Request?',
+          color: 'review',
+          isRendered: this.state.showDialogReviseDAR,
+          showModal: this.state.showDialogReviseDAR,
+          action: { label: "Yes", handler: this.dialogHandlerReviseDAR }
+        }, [div({ className: "dialog-description" }, ["Are you sure you want to revise this Data Access Request?"]),]),
```

Comment:
The color scheme for the "Yes" button is very difficult to read. We should look for a way to either change the non-hover background color so the white text is visible, or change the non-hover text color.

![Screen Shot 2022-01-18 at 8 12 45 AM](https://user-images.githubusercontent.com/116679/149947455-acd1374d-4577-4835-a0ae-2d8d1bee0ecd.png)